management/src/clusterm/manager/api.go

```diff
@@ -141,36 +137,35 @@ func validateAndSanitizeEmptyExtraVars(errorPrefix, extraVars string) (string, e
 	return extraVars, nil
 }
 
-func (m *Manager) nodesCommission(tags []string, sanitizedExtraVars string) error {
-	me := newWaitableEvent(newCommissionEvent(m, tags, sanitizedExtraVars))
+func (m *Manager) nodesCommission(req *APIRequest, extraVars string) error {
+	if !IsValidHostGroup(req.HostGroup) {
```

Comment:
I mean we will continue to fail early by moving this validation to the validate function in event processor.